flexible/mobile-backend/src/main/java/com/google/cloud/solutions/flexenv/common/Message.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Google LLC
```

Comment:
You mentioned you were moving, so this is probably ok, but it seems wrong to checkin a new file w/ this date.

Reply:
Updated